src/work-allocation/containers/available-tasks/available-tasks.component.spec.ts

```diff
@@ -51,7 +52,8 @@ describe('AvailableTasksComponent', () => {
         { provide: LocationDataService, useValue: mockLocationService },
         { provide: Router, useValue: mockRouter },
         { provide: InfoMessageCommService, useValue: mockInfoMessageCommService },
-        { provide: AlertService, useValue: mockAlertService }
+        { provide: AlertService, useValue: mockAlertService },
+        { provide: WorkAllocationFeatureService, useValue: mockFeatureService }
```

Comment:
Doesn't look like this is needed anymore... the constructor doesn't look for it...

Reply:
Removed